packages/application-shell/src/components/project-container/project-container.js

```diff
@@ -28,6 +28,15 @@ const shouldShowNotificationForTrialExpired = daysLeft =>
   daysLeft <= minDaysToDisplayNotification &&
   daysLeft >= maxDaysToDisplayNotification;
 
+export class RedirectToProjectCreate extends React.Component {
```

Comment:
I think we can fix that problem here:
- if it’s not `servedByProxy` we don’t redirect and show a simple message

If we’re in the account app already, there won’t be any redirect and the route will match and let the app handle the rest.

Reply:
Yup I agree. Thanks. Have a good weekend 🤪

Reply:
843ae4c